Useful tools & utilities/Figma.md

```diff
@@ -0,0 +1,124 @@
+collaboration. Similar to Google Docs, team members can easily communicate, collaborate and edit the same file in Figma in real time.
+
+## Benefits of using Figma
+
+- **Simple and intuitive**
+
+Most of us are already very familiar with using tools like Google Docs and Figma isn't much different from that. First-time Figma users can
+get quickly adjusted to using Figma and it won't take them too long to understand most of its functionalities.
+
+- **Easier team collaboration and communication**
+
+Communication and collaboration within Figma is extremely easy and convenient since Figma allows you to leave comments on the designs.
+Discussions can be opened in the comment's thread and if you have the right permissions for it, you could also @mention certain people in
+order to get a faster response from the discussion-relevant team members.
+
+- **Everything you need is at one place**
+
+Screen designs, components, assets, styles, comments, discussions and so much more can be found within a single Figma file. No need to
+include any third-party tools (unless the team feels like they would benefit from them), so basically, for all UI related stuff, you can
+refer to the Figma file. You want it, you open Figma, you got it, no further actions needed. :)
+
+- **Platform flexibility**
+
+If you have any operating system and you run any web browser, you are ready to use Figma. You can share, open, edit Figma files through your
+web browser without any additional setup. The universality of Figma means that the design work will be available to everyone, despite the
+platform they are using.
+
+- **Flexible user permissions management**
+
+Each person can have either a view or an edit access to the Figma files. Additionally, the admin and owner roles can be set accordingly. All
+permissions can be easily managed and customized as desired. For a more detailed description about Figma team permissions and what different
+access modes entail, check out the following [link](https://help.figma.com/hc/en-us/articles/360039970673-Team-permissions).
+
+## Figma essentials
+
+Let's take a look at some of the things which you'll probably be using very frequently as a developer.
+
+### Frames
+
+![Figma Frames](/img/figma/figma_frames.png "Frames in Figma")
+
+Frames serve as top-level containers for most elements in Figma. Usually, each screen has a separate frame and they are all listed
+on the left side as shown in the image above. In this example project, the frames are located in the Layers tab, however, keep in
+mind that this is customizable and the tab naming in your project might differ from this project's conventions (for example, you might have
+a Pages tab rather than a Layers tab). What's important is to know the fundamentals and what Figma provides as functionalities and then,
+minor things like naming conventions and way of structuring stuff within the team is easily manageable.
```

Comment:
like naming conventions and way of structuring stuff within the team **are** easily manageable